### PR TITLE
man: add crosslink between systemd.resource-control(5) and systemd.exec(5)

### LINKED ISSUE #4379: systemd.resource-control manual page should be renamed
### Submission type
- [ ] Bug report
- [x] Request for enhancement (RFE)

_NOTE: Do not submit anything other than bug reports or RFEs via the issue tracker!_
### systemd version the issue has been seen with

> 231

_NOTE: Do not submit bug reports about anything but the two most recently released systemd versions upstream!_
### Used distribution

> Ubuntu 16.10
### In case of bug report: Expected behaviour you didn't see

> …
### In case of bug report: Unexpected behaviour you saw

> …
### In case of bug report: Steps to reproduce the problem

> There are documented options for resource control in _systemd.exec_ too like **IOSchedulingClass**. Yes, I know **Control Groups** are mentioned in the manual page but still I find current name to be misleading. Maybe _systemd.cgroups-control_?

### PR DESCRIPTION
**Pull request by keszybz**
Fixes #4379.

### CODE REVIEW

File: man/systemd.resource-control.xml [hunk 1]
```diff
@@ -83,6 +83,11 @@
     [Slice], [Scope], [Service], [Socket], [Mount], or [Swap]
     sections, depending on the unit type.</para>
 
+    <para>In addition, options which control resources available to programs
+    <emphasis>executed</emphasis> by systemd are listed in
+    <citerefentry><refentrytitle>systemd.slice</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
```
Comment (ghost) [inline man/systemd.resource-control.xml]: I think **systemd.exec** instead of **systemd.slice**. @keszybz

Comment (lucaswerkmeister) [inline man/systemd.resource-control.xml]: Shouldn’t this be `systemd.exec` instead of `systemd.slice`?

Comment (keszybz) [reply man/systemd.resource-control.xml]: Thanks, fixed!

File: man/systemd.exec.xml [hunk 2]
```diff
@@ -74,6 +74,11 @@
     execution specific configuration options are configured in the
     [Service], [Socket], [Mount], or [Swap] sections, depending on the
     unit type.</para>
+
+    <para>In addition, options which control resources through cgroups
```
Comment (poettering) [inline man/systemd.exec.xml:78]: i am pretty sure we should avoid using the term "cgroups" like this. In prose we should spell this out as "Control Groups" or so.

**Pull request merged**